scipy/linalg/flapack_sym_herm.pyf.src

```diff
@@ -1410,3 +1410,40 @@ subroutine <prefix2c>heequb(lower,n,a,lda,s,scond,amax,work,info)
     integer intent(out) :: info
 
 end subroutine <prefix2c>heequb
+
+
+subroutine <prefix>sytri(lower,n,a,lda,ipiv,work,info)
+    ! ?SYTRI computes the inverse of a symmetric indefinite matrix
+    ! A using the factorization A = U*D*U**T or A = L*D*L**T computed by
+    ! ?SYTRF.
+    threadsafe
+    callstatement (*f2py_func)((lower?"L":"U"),&n,a,&lda,ipiv,work,&info)
+    callprotoargument char*, F_INT*, <ctype>*, F_INT*, F_INT*, <ctype>*, F_INT*
+
+    integer optional intent(in), check(lower==0||lower==1) :: lower = 0
+    integer intent(hide), depend(a) :: n=shape(a,1)
+    <ftype> dimension(n,n),intent(in,out,copy,out=inv_a) :: a
+    integer depend(n),intent(hide) :: lda = max(n,1)
```

Comment:
In LAPACK function conventions, lda is the number of rows and n is the number of columns. So typically they are not dependent. This is to allow for passing 20x10 array and be able to solve 5x5 corner of it where n=5 defines the size of the problem and lda tells you how many items to skip for the next column. Hence a size should indeed be `(lda, n)` and `lda = max(1, shape(a, 0))`.

Reply:
Thanks for the explanation! Have changed it